templates/interfaces_nat.j2

```diff
@@ -7,12 +7,15 @@ iface lxcbr0 inet static
     bridge_ports none
     address {{ lxcbr0_ip }}
     netmask {{ lxcbr0_netmask }}
-    post-up iptables -A FORWARD -i lxcbr0 -s {{ lxcbr0_ip }}/{{ lxcbr0_netmask }} -j ACCEPT
-    post-up iptables -A POSTROUTING -t nat -s {{ lxcbr0_ip }}/{{ lxcbr0_netmask }} -j MASQUERADE
+    post-up   iptables -A FORWARD -i $IFACE -s {{ lxcbr0_ip }}/{{ lxcbr0_netmask }} -j ACCEPT
```

Comment:
I updated commits messages. Can you review then and maybe precise me unclear details ? I'll update them to be more understandable.